Octokit/Clients/ITeamsClient.cs

```diff
@@ -13,6 +13,11 @@ namespace Octokit
     /// </remarks>
     public interface ITeamsClient
     {
+        /// <summary>
+        /// Returns a client to manage discussions in a team.
+        /// </summary>
+        ITeamDiscussionsClient TeamDiscussion { get; }
```

Comment:
We want to access this API like `client.Team.Discussion.Get()` so this property name should just be `Discussion`

```suggestion
        ITeamDiscussionsClient Discussion { get; }
```